libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKUpgradeManager.java

```diff
@@ -115,7 +109,9 @@ protected synchronized void upgradeAccMgr() {
             	upgradeTo2Dot2();
             }
         } catch (NumberFormatException e) {
-        	Log.e("UpgradeManager:upgradeAccMgr", "Failed to parse installed version.");
+            SalesforceLogger.getLogger(SalesforceSDKManager.SF_SDK_COMPONENT_NAME,
```

Comment:
Or could we go further and have a SalesforceSDKManager aware helper class with static methods for info/warning/error, so the call would look like
```java
SDKLogger.e(SF_SDK_COMPONENT_NAME, TAG, ...)
```